tools/obscuroscan_v3/frontend/src/hooks/useBatches.ts

```diff
@@ -0,0 +1,19 @@
+import { getBatches, getLatestBatch } from "@/api/batches";
+import { useQuery } from "@tanstack/react-query";
+import { pollingInterval } from "../lib/constants";
+
+export const useBatches = () => {
```

Comment:
While I understand `useBatches` naming can it be updated so it's clear what it does ?